packages/blocks/src/page-block/edgeless/components/component-toolbar/release-from-group-button.ts

```diff
@@ -0,0 +1,35 @@
+import '../buttons/tool-icon-button.js';
+
+import { WithDisposable } from '@blocksuite/lit';
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { ReleaseFromGroupButtonIcon } from '../../../../icons/index.js';
+import type { SurfaceBlockComponent } from '../../../../surface-block/surface-block.js';
+
+@customElement('edgeless-release-from-group-button')
+export class EdgelessReleaseFromGroupButton extends WithDisposable(LitElement) {
+  @property({ attribute: false })
+  surface!: SurfaceBlockComponent;
+
+  protected override render() {
+    return html`<edgeless-tool-icon-button
+      .iconContainerPadding=${2}
+      @click=${() => {
+        this.surface.group.releaseFromGroup(
+          this.surface.edgeless.selectionManager.firstElement
+        );
+      }}
+      .tooltip=${'Release From Group'}
```

Comment:
Note the toolbar button font size:

<img width="545" alt="image" src="https://github.com/toeverything/blocksuite/assets/7312949/415b9a60-89dd-451f-8df0-3b87dd3f8b95">